### PR TITLE
Reverse array

### PR DESCRIPTION
**Pull request by jadhavrajashree40**

### CODE REVIEW

File: PracticeExample/Circle.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package PracticeExample;
+
+public class Circle {
+	int radius;
+	float PI = 3.14f;
+
+	Circle(int r) {
```
Comment (amitsahu8) [inline PracticeExample/Circle.java:7]: use public or private in method

File: PracticeExample/EvenOdd_Number.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package PracticeExample;
+
+import java.util.Scanner;
+
+public class EvenOdd_Number {
+	public static void main(String[] args) {
+		for (int n = 1; n <= 30; n++) {
+			System.out.println("Enter the number");
+			Scanner sc = new Scanner(System.in);
```
Comment (amitsahu8) [inline PracticeExample/EvenOdd_Number.java:9]: remove scanner

File: PracticeExample/EvenOdd_Number.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+			int n1 = sc.nextInt();
+			if ((n1 % 2 != 0) && (n1 >= 1 && n1 <= 30)) {
+				System.out.println(n1 + "Weird");
```
Comment (amitsahu8) [inline PracticeExample/EvenOdd_Number.java:12]: output message should be meaningful

File: PracticeExample/Test.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package PracticeExample;
+
+public class Test {
+	public static void main(String[] args) {
+		Cylinder c1 = new Cylinder(4, 2);
```
Comment (amitsahu8) [inline PracticeExample/Test.java:5]: use meaningful variable name and not c1, c

File: PracticeExample/ReverseArray.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package PracticeExample;
+
+public class ReverseArray {
+	public static void main(String[] args) {
+		int arr[] = { 1, 3, 5, 7, 9, 11, 15, 17, 19 };
+		int size = 3;
+		int n = arr.length;
```
Comment (amitsahu8) [inline PracticeExample/ReverseArray.java:7]: what is n?

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: PracticeExample/EvenOdd_Number.java (hunk 2 above)
Comment (jadhavrajashree40) [reply PracticeExample/EvenOdd_Number.java:9]: scanner is removed

File: PracticeExample/EvenOdd_Number.java (hunk 3 above)
Comment (jadhavrajashree40) [reply PracticeExample/EvenOdd_Number.java:12]: its done

File: PracticeExample/Circle.java (hunk 1 above)
Comment (jadhavrajashree40) [reply PracticeExample/Circle.java:7]: use public method.

File: PracticeExample/Test.java (hunk 4 above)
Comment (jadhavrajashree40) [reply PracticeExample/Test.java:5]: Adding meaningful name.

File: PracticeExample/ReverseArray.java (hunk 5 above)
Comment (jadhavrajashree40) [reply PracticeExample/ReverseArray.java:7]: changed variable name

**Pull request closed without merging**